Packs/Axonius/Integrations/Axonius/Axonius.py

```diff
@@ -129,7 +129,7 @@ def command_results(
 ) -> CommandResults:  # noqa: F821, F405
     """Parse assets into CommandResults."""
     api_name = api_obj.__class__.__name__
-    aql = api_obj._LAST_GET["filter"]
+    aql = api_obj.LAST_GET["filter"]
```

Comment:
Suggests in any case to change `.LAST_GET["filter"]` to `LAST_GET.get("filter")`
Our past experience thought us that you can never know 😃

Reply:
I can be ok with making that change - although i create that dictionary myself so i can definitely know it will be there